docker-compose.lab.yml

```diff
@@ -11,7 +11,7 @@ services:
         volumes:
             - aiida-postgres-db:/var/lib/postgresql/data
         healthcheck:
-            test: ["CMD-SHELL", "pg_isready"]
+            test: ["CMD-SHELL", "pg_isready -U $${POSTGRES_USER}"]
```

Comment:
Why was this not needed before?

Reply:
Because this is not a critical issue, without this it will pop a warning as discussed https://github.com/peter-evans/docker-compose-healthcheck/issues/16, the healthcheck still pass somehow.